backend/question-service/src/controllers/questionController.ts

```diff
@@ -34,6 +36,9 @@ export const createQuestion = async (
       description,
       complexity,
       category,
+      testcases,
+      testcaseInputFileUrl,
+      testcaseOutputFileUrl,
```

Comment:
![image](https://github.com/user-attachments/assets/63d99c44-bfcc-4e78-9316-7423a4e5b44f)

testcases is to be displayed on frontend for user to see like in the figma (admin can add up to 3) and especially if there's more than 1 input variable so can do sth like `nums = [3, 4, 5], k = 1` 

the urls is to upload the file version of the testcases, can add as many as the admin wants within the file (so can potentially have hidden test cases + easier to scale) and also may not be as user-friendly bc the input variables might just be delimited by a comma or newline (without the `nums = ` or `k = `) 

that's why i separated them but if there's a more elegant way of doing this can j lmk

Reply:
i think the figma is a little outdated, as it follows more of leetcode style (so in your example, nums will be a list, and target will be an integer, which will be the parameters for the function).

but i think in our case we follow more of kattis style (where we use the standard input instead). So i think using the files to display the test cases should be be clearer instead because the user should know exactly how the input format look like instead?

Reply:
ohh ok got it 👍